### PR TITLE
Basic test cases

### PR DESCRIPTION
**Pull request by forslund**
The bare minimum so far.

### CODE REVIEW

File: test/intent/sample1.intent.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
-  "utterance": "alarm in 2 months",
-  "intent_type": "AlarmSkillCreateIntent",
-  "intent": {
-    "AlarmSkillCreateVerb": "alarm"
+  "utterance": "set an alarm for 6 pm",
+  "evaluation_timeout": 10,
```
Comment (MatthewScholefield) [inline test/intent/sample1.intent.json]: Is there a default evaluation timeout? Just wondering if there's a way to skip putting it on every test case.

**Review by MatthewScholefield — CHANGES_REQUESTED** [review]

File: test/intent/sample1.intent.json (hunk 1 above)
Comment (forslund) [reply test/intent/sample1.intent.json]: the default is 30 seconds. It is not needed at all and should be removed. Thanks for noticing will fix.

File: test/intent/sample3.intent.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
-  "intent": {
-    "AlarmSkillCreateVerb": "timer"
-  }
-}
\ No newline at end of file
+  "utterance": "set an alarm for 6 am every weekday",
+  "evaluation_timeout": 10,
+  "expected_data": {
+      "ampm": "am",
+      "time": "6"
```
[the remark this answers is not in the document]
Comment (forslund) [reply test/intent/sample3.intent.json]: yes

**Comment by forslund** [comment]
PR updated

**Comment by forslund** [comment]
Ok, I'm merging these changes now.

**Pull request merged**